### DISCUSSION TITLE
OK, I have a beta build that I want my patrons to test. I took the supermium patch, removed the supermium branding and a few flags, as well as removed the recently added windows xp compatibility code (because it doesn't work yet, isn't ready for prime time, and causes a compiler warning on windows 7). I then added Thorium's patches that have to do with compiler optimizations and speed (but none of the others)  Essentially it is a compiler optimized vanilla build of Chromium with a few flags (including the directx vs GDI flag, so that you can test). I will be uploading it to the repo, except it will be hidden behind the beta flag. I simply want you guys to daily drive it for a day or two and report any weird behaviour or crashes.

### DISCUSSION THREAD

**Discussion by Alex313031** [Alex313031/thorium-legacy#32] [Announcements]
OK, I have a beta build that I want ya'll to test.
I took the supermium patch, removed the supermium branding and a few flags, as well as removed the recently added windows xp compatibility code (because it doesn't work yet, isn't ready for prime time, and causes a compiler warning on windows 7). I then added Thorium's patches that have to do with compiler optimizations and speed (but none of the others).

Essentially it is a compiler optimized vanilla build of Chromium with a few flags (including the directx vs GDI flag, so that you can test that). Please report any weird behavior or crashes.

<hr /><em>This discussion was created from the release <a href='https://github.com/Alex313031/thorium-win7/releases/tag/'>OK, I have a beta build that I want my patrons to test. I took the supermium patch, removed the supermium branding and a few flags, as well as removed the recently added windows xp compatibility code (because it doesn't work yet, isn't ready for prime time, and causes a compiler warning on windows 7). I then added Thorium's patches that have to do with compiler optimizations and speed (but none of the others)  Essentially it is a compiler optimized vanilla build of Chromium with a few flags (including the directx vs GDI flag, so that you can test). I will be uploading it to the repo, except it will be hidden behind the beta flag. I simply want you guys to daily drive it for a day or two and report any weird behaviour or crashes.</a>.</em>

**Comment by libreom** [comment]
Thanks

**Comment by verifyurhuman** [comment]
will you collaborate with the author of supermium? I mean, you can help him with his browser project and he can help with yours

**Comment by libreom** [comment]
Please also provide sse4 or 3 build as my pc doesn't support avx

**Comment by Alex313031** [comment]
Yes, we are collaborating.

**Comment by Alex313031** [comment]
Sorry, but non-avx builds will come with the final M119 release.  (ANY DAY
NOW)
